templates/repo/issue/new_form.tmpl

```diff
@@ -140,7 +140,7 @@
 		</div>
 		{{end}}
 		<div class="divider"></div>
-			<input id="assignee_ids" name="assignee_ids" type="hidden" value="{{.assignee_ids}}">
```

Comment:
I think `.assignee_ids` has been used in repo_form.go:445? Can we ignore it directly?

Reply:
I renamed this variable and continued using `assignee_ids`. See de17b5ee5b35491d860b380b9f3cef2fe77fb312